src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -3564,23 +3580,48 @@ module Util =
             traitsToRender
             |> List.collect (fun (isNonInterface, tMethods, tEntRef) ->
                 let tEnt = com.GetEntity(tEntRef)
+                let memberItemsStatic =
+                    decl.AttachedMembers
+                    |> List.filter (fun m ->
+                        let isDynamic = m.Args |> List.exists (fun q -> q.IsThisArgument)
+                        tMethods |> Set.contains m.Name && not isDynamic)
+                    |> List.map (makeMemberItem ctx)
                 let memberItems =
                     decl.AttachedMembers
-                    |> List.filter (fun m -> tMethods |> Set.contains m.Name)
+                    |> List.filter (fun m ->
+                        let isDynamic = m.Args |> List.exists (fun q -> q.IsThisArgument)
+                        tMethods |> Set.contains m.Name && isDynamic)
                     |> List.map (makeMemberItem ctx)
-                if List.isEmpty memberItems then
-                    []
-                else
+
+                let implItemStatic =
                     let ty =
                         let genArgs = getEntityGenArgs ent
                         let genArgTys = genArgs |> transformGenArgs com ctx
                         let bounds = mkTypeTraitGenericBound [entName] genArgTys
                         let ty = mkTraitTy [bounds]
-                        // TODO: Wrapping the impl block in a Lrc breaks casting (see IEnumerable), because it no longer implements the interface for T but for Lrc<T>
-                        // let entTyp = Fable.DeclaredType(entRef, genArgs)
-                        // maybeWrapInPtrTy com ctx entTyp ty
+                        let ety = com.GetEntity(tEntRef)
```

Comment:
@alexswan10k This doesn't seem to be used.

Reply:
Good spot, fixed